src/components/ClientEvents.vue

```diff
@@ -1,6 +1,113 @@
 <template>
   <section class="fields">
-    <h1>Events</h1>
+    <h1>Client Milestones</h1>
+    <div>
+      <div class="columns">
+        <div class="column">
+          <div class="field">
+            <button class="primary" @click="$router.push('/events/add')">
+              Add Event
+            </button>
+          </div>
+
+          <div class="card">
+            <div class="fields">
+              <h1>Welcome Email with steps</h1>
+              <!-- Event name -->
+              <div class="field">
+                <p>Last updated on:</p>
+                <input v-model="lastUpdated" type="date" style="width:40%" />
+                <button class="primary" style="margin-left:5px">Save</button>
+              </div>
+              <div>
+                <label> Status: </label>
+                <select v-model="selected" style="width:40%">
```

Comment:
Maybe give a default value to each `<select>` to indicate the status each client starts with.

Reply:
Added the <selected> attribute to "Not Done" status, however not updating when we run the code

Reply:
So you need to have a spot in `data` for storing the current state of each `<select>`, and bind `<select>` with that data. At the same time, assign a value to each `<option>` (the content wrapped within `<option></option>` doesn't automatically become its value). When the item being selected is updated, update the value in `data`.

Reply:
Take a look: https://vuejs.org/v2/guide/forms.html#Select

Reply:
So you need to have a field in `data` for storing the current state of each `<select>`, and bind `<select>` with that field using `v-model`.

Take a look: https://vuejs.org/v2/guide/forms.html#Select